### PR TITLE
Add coala organization

### PR DESCRIPTION
**Pull request by li-boxuan**
Related to https://github.com/gsoc-cn/gsoc-cn/issues/20

### CODE REVIEW

File: resources/organizations/coala.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# coala
+
+## 基本介绍
+
+[coala][1] 是一个 Python 实现的命令行 linter，提供一个统一的命令行接口，用于代码分析和修复，无论你使用的是哪种开发语言。官方的一句话介绍是：linting and fixing code for all languages。同时，coala支持多种IDE/编辑器的扩展。coala也可以用在Continuous Integration上，比如自动检查github上的Pull Request，并自动生成修复的patch。
+
+以下内容节选自[开源项目哪里找？ - 高策的回答 - 知乎][2](有改动)，主要是推荐coala社区的理由：
+
+> 1. 社区有完整的贡献指南和从newcomer到developer再到maintainer的贡献路线，对新人非常友好;
+2. 社区有完善的CI/CD 流程，以及文档，基础设施完整且实用;
```
Comment (gaocegege) [inline resources/organizations/coala.md]: 每一行都需要 `>` 哦

**Comment by gaocegege** [comment]
Thanks for your awesome work! Please fix the nits and I will merge is ASAP.

File: resources/organizations/coala.md (hunk 1 above)
Comment (li-boxuan) [reply resources/organizations/coala.md]: Lol it works well on my MacDown editor. I'll fix it in a minute.

Comment (li-boxuan) [reply resources/organizations/coala.md]: Done.

**Comment by gaocegege** [comment]
LGTM

**Pull request merged**